datajoint/blob.py

```diff
@@ -71,6 +72,9 @@ def __init__(self, squeeze=False):
         self.protocol = None
 
     def set_dj0(self):
+        if not config.get('enable_python_native_blobs'):
+            raise DataJointError('v0.12+ python native blobs disabled. see also: https://github.com/datajoint/datajoint-python/blob/master/README.md')
```

Comment:
```suggestion
            raise DataJointError('v0.12+ python native blobs disabled, see the 0.12.0 release notes at: https://github.com/datajoint/datajoint-python/blob/master/README.md')
```

Reply:
Well.. these aren't actually the release notes, but a special section in the README.
alternate suggestion?

Reply:
Where is it? Are you planning to add in this PR?

Reply:
It's in the docs-parts PR - can merge the pr's if that's better

Reply:
ok, merging and will sort in the next PR